cxx4opencl/diff2openclc.txt

```diff
@@ -4,17 +4,21 @@
 
 === Difference to OpenCL C
 
-{cpp} for OpenCL aims to achieve backwards compatibility with OpenCL C
-for the majority of features. However, it is a different
-language that is derived from {cpp} and therefore it inherits
-its fundamental design principles. Hence {cpp} for OpenCL
-deviates from OpenCL C in the same areas where {cpp} deviates
-from C.
+{cpp} for OpenCL provides backwards compatibility with OpenCL C
+for the majority of features. However, there are a number of
+exceptions that are described in this section. Some of them come
+from the nature of C++ but others are due to improved behaviour
+of a kernel language.
```

Comment:
This reads a bit strange, do you mean to say improvements that have been added just for C++ for OpenCL?

Reply:
I changed it now as follows:

>  Some of them come
> -from the nature of C++ but others are due to improved behaviour
> -of a kernel language.
> +from the nature of C++ but others are due to improvements in OpenCL
> +features. Most of such improvements do not invalidate old code,
> +but simply provide extra functionality.